examples/features/torch_batch_process/batch_inference/compare_with_core_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Batch inference with Core API & Torch Batch Processing API
+## Overview
+This example illustrates how to run distributed batch inference with Core API. Determined's Core API is very flexible
+and can be used to run almost anything, including batch inference. 
+
```

Comment:
this would be inconsistent with the [published docs](https://docs.determined.ai/latest/model-dev-guide/apis-howto/overview.html). 

> The Core API is a low-level, flexible API that lets you train models in any deep learning framework. With the Core API, you can plug in your existing training code. 

Not sure which one we want to edit, or both?

Reply:
Both are technically true statements.

The existing doc could be modified to say "core api can do more than just training" but since the core API is still mostly only enabled in hpsearch jobs (that is, "experiments"), it wouldn't be that meaningful of an edit.  Our inference MVP is sort of hacking an "experiment" (aka an hp search experiment) for inference purposes.

So if it were up to me, I would leave these statements alone, and modify the currently-published document when we support non-training tasks in a more first-class way.